content/Watsonx/WatsonxAI/102.md

```diff
@@ -0,0 +1,250 @@
+    | T2 | 0.15 |
+    | T3 | 0.14 |
+    | T4 | 0.13 |
+    | T5 | 0.12 |
+    | T6 | 0.11 |
+    | T7 | 0.10 |
+    | T8 | 0.09 |
+
+    In this case when you set:
+
+    - **Top P = 0.5**, the model chooses randomly among T1, T2, T3, and T4 (because the probabilities sum up to > 0.5).
+
+    - **Top K = 50**, the model chooses randomly among all 8 of the tokens (as 8 < 50)
+
+    Thus, for this case, you need to lower **Top K** to **4** to have a similar effect as **Top P = 0.5** so both settings are choosing among the top 4 tokens.
+
+
+## Prompting for different output formats
+
+When prompting an LLM, it's often beneficial to get the output in a specific format. For readability, or for streamlining an integration to another tool. 
+
+### Prompting for Lists
+
+Remember that a foundation model does not read and interpret a prompt like a human. Rather, it consumes a series of tokenized words and calculates the best next token. The concept of a list is easy for a human to understand. Here we will see how some LLMs respond. 
+
+1. [Open a new Prompt Lab session](/watsonx/watsonxai/000#creating-a-new-prompt-lab-session) and select the **Freeform** tab instead of the default **Structured** tab. Choose "Switch mode" when prompted to Switch to freeform mode.
+
+    > Note: In Freeform mode, you still have access to all the examples, models, and configuration, but there is no structural guide for the prompt.
+
+2. Open the **Model parameters** panel.  In **Freeform** mode, the parameter defaults look like the following if you have **not selected** a prompt sample.
+
+    ![greedy_params](./images/102/greedy-parameters.png)
+
+3. Verify that you are using **Greedy** decoding, and change the value of **Max tokens** to 100. 
+
+4. Ensure that you are using the **flan-ul2-20b** model.  Copy and paste the following text in the input field:
+
+    ```txt
+    The following paragraph is a consumer complaint. 
+
+    Read the following paragraph and list all the issues.   I called your helpdesk multiple times and every time I waited 10-15 minutes before I gave up. The first time I got through, the line got cut suddenly and I had to call back. This is just ridiculous. When I finally got through like after 3 days (yes, 3 days) your agent kept going over a long checklist of trivial things and asking me to verify, after I repeatedly told the agent that I am an experienced user and I know what I am doing. It was a complete waste of time. After like an eternity of this pointless conversation, I was told that an SME will contact me. That – was 2 days ago. What is the problem with your support system?
+
+    The list of issues is as follows:
+    ```
+
+    Click **Generate**.  You should see output similar to the following:
+
+    ![list_output](./images/102/list-output.png)
+
+    **Note the following:**
+    - The model did not output a list.
+    - The model just repeated the input text word by word.
+    - The output terminates due to the 100 token limit
+
```

Comment:
4th line looks like part of 3rd bullet points. Add another line space to differentiate it.

Reply:
👍 Will address in subsequent PR, thanks!